### ISSUE TITLE
Consider if asserts should/can have debug messages

### ISSUE THREAD

**Issue by AdamKorcz** [theupdateframework/tuf-conformance#213]
We can consider if we should attach messages to asserts that explain the likely problem it fails and why it should not fail. This can help clients save time, and client developers will need a lower degree of understanding of the internals of the conformance suite to use it; They won't need to know how we develop our tests to understand a failure.

**Comment by jku** [comment]
Yeah this would make sense at least in cases where we currently do something like this 

```
# comment explaining why we assert x
assert x
```

this could be just 

```
assert x, "explanation here"
```

I assume pytest would show that in a reasonable way
